device-types/Cisco/MDS-C9148T.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148T
+slug: ds-c9148t
+part_number: DS-C9148T-K9
```

Comment:
If you look at [this datasheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148t-32-gbps-48-port-fibre-channel-switch/data-sheet-c78-740623.html#Formoreinformation) which specific model number is this? I see the difference between table 3 & 4 is just the fan configuration, but what about table 5?

Reply:
Updated the part number, file name, slug, and created an additional .yaml file for the port side intake model (DS-C9148T-48PETK9.yaml).
Changed file name from DS-C9148T to DS-C9148T-48PITK9
manufacturer: Cisco
model: MDS 9148T 48X32G FC
slug: cisco-ds-c9148t-48pitk9
part_number: DS-C9148T-48PITK9
u_height: 1
is_full_depth: true
airflow: front-to-rear

New file name DS-C9148T-48PETK9
manufacturer: Cisco
model: MDS 9148T 48X32G FC
slug: cisco-ds-c9148t-48petk9
part_number: DS-C9148T-48PETK9
u_height: 1
is_full_depth: true
airflow: rear-to-front